wordpress-mobile/WordPress-Android
# WP Stories integration phase 1

This PR implements integration for WordPress Stories with a git submodule.

#### To test:
1. checkout the branch and run 
```
git submodule init 
git submodule update
```
or for a fresh clone, you can also do `git clone --recurse-submodules git@github.com:wordpress-mobile/WordPress-Android.git`

2. make sure the `libs/portkey-android` is populated
3. build
4. run

PR submission checklist:
